src/client/pkg/grpcutil/stream.go

```diff
@@ -137,3 +137,9 @@ func WriteFromStreamingBytesClient(streamingBytesClient StreamingBytesClient, wr
 	}
 	return nil
 }
+
+// CopyStreamingBytes copies bytes from a StreamingBytesClient (src) to a
+// StreamingBytesServer (dst).
+func CopyStreamingBytes(dst StreamingBytesServer, src StreamingBytesClient) error {
+	return WriteFromStreamingBytesClient(src, NewStreamingBytesWriter(dst))
```

Comment:
I'm not sure why this declaration is needed ... it's about as handy as the underlying call? Also ... this method doesn't seem to be called anywhere.

Reply:
I was using this at one point but got rid of the call and just used the method above. I'll get rid of it, since I don't think it's really going to be useful to anyone.